src/CMakeLists.txt

```diff
@@ -589,7 +589,7 @@ IF(APPLE)
 	  )
      ENDIF()
      # workaround for https://bugreports.qt.io/browse/QTBUG-105984
-     IF(ENABLE_MEDIA AND (${QT_VERSION} VERSION_GREATER_EQUAL "6.4.0"))
+     IF(ENABLE_MEDIA AND (${QT_VERSION} VERSION_EQUAL "6.4.0"))
```

Comment:
QTBUG-105984 was fixed in 6.4.1 and 6.5.0

Reply:
OK